### PR TITLE
disabled checking for updates if in NH

### PR DESCRIPTION
**Pull request by JulianoGianlupi**

### CODE REVIEW

File: cc3d/player5/ViewManager/SimpleViewManager.py [hunk 1]
```diff
@@ -453,6 +453,11 @@ def check_version(self, check_interval=-1, display_no_update_info=False):
         :return:None
         """
 
+        from os import environ
```
Comment (CompuCell3D) [inline cc3d/player5/ViewManager/SimpleViewManager.py:456]: move import to the top of the file

**Review by CompuCell3D — CHANGES_REQUESTED** [review]

File: cc3d/player5/ViewManager/SimpleViewManager.py (hunk 1 above)
Comment (CompuCell3D) [reply cc3d/player5/ViewManager/SimpleViewManager.py:456]: otherwise it looks good!

**Review by tjsego — APPROVED** [review]

File: cc3d/player5/ViewManager/SimpleViewManager.py (hunk 1 above)
Comment (JulianoGianlupi) [reply cc3d/player5/ViewManager/SimpleViewManager.py]: done

**Comment by JulianoGianlupi** [comment]
@CompuCell3D @tjsego can we also push this change to v4.2.4? that way NH will get the pertinent update sooner

**Pull request merged**